ntp.go

```diff
@@ -114,6 +114,7 @@ type Response struct {
 	ReferenceID    uint32        // server's reference ID
 	RootDelay      time.Duration // server's RTT to the reference clock
 	RootDispersion time.Duration // server's dispersion to the reference clock
+	Leap           uint8         // server's leap second indicator 1 means increase 1 second, etc.
```

Comment:
Can you fix comment? Given that 0 = no warning, 1 = "last minute has 61 seconds", 2 = "last minute has 59 seconds" and 3 = "unknown", the comment is unclear.

How about making the comment "server's leap second indicator; see RFC 5905"